build.rs

```diff
@@ -23,27 +23,22 @@
 
 extern crate cc;
 
-use std::io::{self, Write};
+use std::env;
 
 fn main() {
     // Check whether we can use 64-bit compilation
-    #[cfg(target_pointer_width = "64")]
-    let use_64bit_compilation = {
+    let use_64bit_compilation = if env::var("CARGO_CFG_TARGET_POINTER_WIDTH").unwrap() == "64" {
```

Comment:
Can you elaborate on where these are different values? Why would target_pointer_width != CARGO_CFG_TARGET_POINTER_WIDTH and isn't that a bug in cargo/rustc?